src/app.res

```diff
@@ -17,46 +17,46 @@ let filterConversations = (
             true
           } else {
             open Utils
-            string_contains(c.name |> String.lowercase, filterText) ||
-            (string_contains(c.email |> String.lowercase, filterText) ||
-            (string_contains(c.phone->getWithDefault("") |> String.lowercase, filterText) ||
-            (string_contains(c.city->getWithDefault("") |> String.lowercase, filterText) ||
-            (string_contains(c.zipcode->getWithDefault("") |> String.lowercase, filterText) ||
-            (string_contains(c.street->getWithDefault("") |> String.lowercase, filterText) ||
-            (string_contains(c.latest_message.content |> String.lowercase, filterText) ||
+            string_contains(String.toLowerCase(c.name), filterText) ||
+            (string_contains(String.toLowerCase(c.email), filterText) ||
+            (string_contains(String.toLowerCase(c.phone->getWithDefault("")), filterText) ||
+            (string_contains(String.toLowerCase(c.city->getWithDefault("")), filterText) ||
+            (string_contains(String.toLowerCase(c.zipcode->getWithDefault("")), filterText) ||
+            (string_contains(String.toLowerCase(c.street->getWithDefault("")), filterText) ||
+            (string_contains(String.toLowerCase(c.latest_message.content), filterText) ||
             string_contains(c.notes, filterText)))))))
           }
         )
 
   switch folder {
   | All =>
     conversations->Js.Array2.filter(c =>
-      !c.is_in_trash || List.exists((c_id: int) => c_id == c.id, interactedWithConversations)
+      !c.is_in_trash || interactedWithConversations->List.has(c.id, (a, b) => a == b)
```

Comment:
Here, `interactedWithConversations` should also just be an array. Then use `Array.some` to check if an item is in the array list.